app/controllers/agent/ManageAgentController.scala

```diff
@@ -190,4 +190,15 @@ class ManageAgentController @Inject()(
         backLink = controllers.agent.routes.ManageAgentController.manageAgent(Some(agentCode)).url
       ))
 
+  private def joinOldRevokeAppointJourney(agentCode: Long) =
+    Redirect(
+      controllers.agentAppointment.routes.AppointAgentController.selectAgentPropertiesSearchSort(
```

Comment:
so you are not making any changes to that old screen?

Reply:
how will the choices in that screen be reflected on the new endpoint?

Reply:
Karen said we would just join the old journey for the "some" options. These. will still go to the old endpoint. We need to confirm with Karen if that is the case.